README

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
This originates from [the blog plugin](https://github.com/rambo/cmsplugin-andersblog) which is used as a reference for the packaging. I'll remove it if there is no need for this.